### DISCUSSION TITLE
utils.py for new Firmware 1.0 Beta

### DISCUSSION THREAD

**Discussion by peregrines2** [Louisvdw/dbus-serialbattery#617] [General]
Hi!

Can somebody please help me out and post your utils.py for new Firmware 1.0 Beta for a 48V lifepo4 16s JK BMS configuration?
I really want to double check and am afraid to set wrong parameters.
Thank You!!!

**Comment by wormsman** [comment]
You do not have to change the utils.py in the new version.  Now it's all managed in the config. When you don't have any specials the new version should work out of the box. The parameters are good to go for an LFP pack and with JK BMS the battery Ah gets read from the BMS. Maybe you have to set the maximum charging and discharging in the config ( see the names in the default config file).

**Reply by peregrines2** [reply]
Thank You so much wormsman!!!
